### PR TITLE
fix(RC): delete message is not sent

### PR DESCRIPTION
**Pull request by MohamadJaara**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

there are 2 issues:
1. current time is calculated incorrectly when deciding if a message should be deleted
2. Message option is set to ignore all when passing a list of specific clients. 

### Solutions

1. calculate the time correctly.
2. add a flag to not ignore all when sending to a specific users.

### Dependencies (Optional)

_If there are some other pull requests related to this one (e.g. new releases of frameworks), specify them here._

Needs releases with:

- [ ] GitHub link to other pull request

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

_Briefly describe how this change was tested and if applicable the exact steps taken to verify that it works as expected._

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Review by trOnk12 — APPROVED** [review]

File: logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/message/SessionResetSender.kt [hunk 1]
```diff
@@ -67,11 +67,19 @@ class SessionResetSenderImpl internal constructor(
...
                 status = Message.Status.SENT,
                 isSelfMessage = true
             )
             val recipient = Recipient(userId, listOf(clientId))
-            messageSender.sendMessage(message, MessageTarget.Client(listOf(recipient)))
+            messageSender.sendMessage(
+                message,
+                MessageTarget.Client(
+                    listOf(recipient),
+                    true
```
Comment (MohamadJaara) [inline logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/message/SessionResetSender.kt:80]: i believe this was buggy before and we should handle if there are  clients missing but i left it as true to not change the logic

**Review by vitorhugods — COMMENTED** [review]
Tests?

**Comment by MohamadJaara** [comment]
> Tests?

self deleting messages logic is missing a lot of tests e.g. `EphemeralMessageDeletionHandler` i will try my best next week to cover some of it

**Comment by MohamadJaara** [comment]
Closed because if wrong base branch for the PR

**Pull request closed without merging**